Data_Analysis_Markdown.Rmd

```diff
@@ -0,0 +1,503 @@
+
+}
+
+country.fig.note.brazil <- arrangeGrob(country.fig[["Brazil"]], 
+                         sub = textGrob("Brazil",
+                                        x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.mexico <- arrangeGrob(country.fig[["Mexico"]], 
+                         sub = textGrob("Mexico",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.guatemala <- arrangeGrob(country.fig[["Guatemala"]], 
+                         sub = textGrob("Guatemala",
+                                        x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.colombia <- arrangeGrob(country.fig[["Colombia"]], 
+                         sub = textGrob("Colombia",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.ecuador <- arrangeGrob(country.fig[["Ecuador"]], 
+                         sub = textGrob("Ecuador",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.bolivia <- arrangeGrob(country.fig[["Bolivia"]], 
+                         sub = textGrob("Bolivia",
+                                        x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.peru <- arrangeGrob(country.fig[["Peru"]], 
+                         sub = textGrob("Peru",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+
+country.fig.note.dominican_republic <- arrangeGrob(country.fig[["Dominican Republic"]], 
+                         sub = textGrob("Dominican Republic",
+                                       x = 0, hjust = -1, vjust = 0,
+                                        gp = gpar(fontface = "bold", fontsize = 12)))
+                 
+
+country_grid <- arrangeGrob(country.fig.note.brazil, 
+          country.fig.note.mexico,
+          country.fig.note.guatemala,
+          country.fig.note.colombia,
+          country.fig.note.ecuador,
+          country.fig.note.bolivia,
+          country.fig.note.peru,
+          country.fig.note.dominican_republic,
+          nrow = 4, ncol = 2,
+          main = textGrob("Effects of Skin Color and Other Factors on\n Educational Attainment", 
+                          gp = gpar(fontface = "bold", fontsize = 20)),
+          sub = textGrob("95 Percent C.I. (Design -Effects Based)", 
+                          gp = gpar(fontface = "bold", fontsize = 18)))
+
+```
+
```

Comment:
Cool.  Nice extension.  Would it be interesting to plot the results by coefficient?  In other words, one panel for all the female coefficients?